### DISCUSSION TITLE
is this app abandoned?

### DISCUSSION THREAD

**Discussion by nadamondi** [ahrm/sioyek#1148] [Q&A] [answered]
I see a lot of issues and PRs but last release was years ago. Is the app abandoned or just does not need any updates?

**Comment by ahrm** [comment, marked as answer]
No, I am working on the next version of sioyek which has android and ios support ( an early version of android support is available in the development branch). 
I am also dealing with some real-life issues which has prevented me to work on sioyek in the past 1-2 months, but hopefully I can return to full-force development soon.
